model/k8s/registry.yaml

```diff
@@ -232,3 +232,69 @@ groups:
               value: 'terminating'
               brief: "Terminating namespace phase as described by [K8s API](https://pkg.go.dev/k8s.io/api@v0.31.3/core/v1#NamespacePhase)"
               stability: experimental
+      - id: k8s.container.status.state
```

Comment:
@open-telemetry/semconv-system-approvers sorry I forget, where did we end up with adding mutable resource attributes prior to entities?

Reply:
These are attributes in the registry which will be used as metrics attributes.

We have an issue to better formalize this at https://github.com/open-telemetry/semantic-conventions/issues/1554.

ref: https://github.com/open-telemetry/semantic-conventions/issues/1032#issuecomment-2420399295